axelor-account/src/main/java/com/axelor/apps/account/service/move/MoveValidateServiceImpl.java

```diff
@@ -235,17 +235,25 @@ public void accounting(Move move, boolean updateCustomerAccount) throws AxelorEx
 
     this.checkPreconditions(move);
 
-    if (move.getPeriod().getStatusSelect() == PeriodRepository.STATUS_CLOSED
-        && !move.getAutoYearClosureMove()) {
-      throw new AxelorException(
-          TraceBackRepository.CATEGORY_CONFIGURATION_ERROR,
-          I18n.get(IExceptionMessage.MOVE_ACCOUNTING_FISCAL_PERIOD_CLOSED));
-    }
-
     Boolean dayBookMode =
         accountConfigService.getAccountConfig(move.getCompany()).getAccountingDaybook()
             && move.getJournal().getAllowAccountingDaybook();
 
+    if (move.getPeriod().getStatusSelect() == PeriodRepository.STATUS_CLOSED
+        && !move.getAutoYearClosureMove()) {
+      if ((dayBookMode == null || dayBookMode)
```

Comment:
Can you change dayBookMode's type from Boolean to boolean? 
So you don't have to check if dayBookMode can be null, thank you